Templates/CSharp/Requests/IEntityReferenceRequest.cs.tt

```diff
@@ -17,6 +18,8 @@ namespace <#=this.GetNamespaceName(entity)#>
 
     <#=this.GetEntityReferenceRequestInterfaceDefinition(entity)#>
     {
+<# if (features.CanDelete) { #>
```

Comment:
I'm going to not make this change right now. I fought with myself on this for a while. The templates are hard enough to read that making that change actually makes them harder to read.